echo-pubsub-google/src/main/resources/gcb.jinja

```diff
@@ -6,5 +6,14 @@
     "name": "{{ image.name }}",
     "version": "{{ image.digest }}",
     "type": "docker/image"
-  } {% if not loop.last %} , {% endif %} {% endfor %}
+  } {% if (not loop.last) or ((artifacts.objects.paths | length) > 1) %} , {% endif %} {% endfor %}
+
+  {% set location = artifacts.objects.location %}
+  {% for path in artifacts.objects.paths %}
+    {
+      "reference": "{{ location }}{{ path }}",
+      "name": "{{ location }}{{ path }}",
+      "version": "{{ location }}{{ path }}",
+      "type": "gcs/object"
```

Comment:
just realised cloud build might be able to upload to non-GCS locations. Need to double check and update this accordingly.

Reply:
Confirmed that objects in this struct are GCS-only: https://cloud.google.com/cloud-build/docs/configuring-builds/store-images-artifacts#storing_artifacts_in_gcs_name_short